### ISSUE TITLE
dead link in ICD10/readme.md to WHO ICD10 files

### ISSUE THREAD

**Issue by The-Alchemist** [OHDSI/Vocabulary-v5.0#923]
The instructions at [[ICD10/readme.md] [ICD10/readme.md](ICD10/readme.md)](https://github.com/OHDSI/Vocabulary-v5.0/tree/master/ICD10#sequence-of-actions) say:

> Download the latest ICD-10 version (e.g. ICD-10 2016 version) on the [WHO Website](http://apps.who.int/classifications/apps/icd/ClassificationDownload/DLArea/Download.aspx)

However, that link is dead, and I couldn't find a replacement.  In fact, I don't think http://apps.who.int/ is active anymore.

Any suggestions?

**Comment by TinyRickC137** [comment]
Hello, @The-Alchemist 

Thank you for being so attentive to details. It is indeed no longer available for downloading. 

We updated the link, even though it is no longer helpful, and contacted the WHO. It may take some time for them to reply and we can't guarantee that ICD10 will be made available for download again. 

At the moment I can only suggest using the [ICD10 web browser](https://icd.who.int/browse10/2019/en#/).

**Comment by TinyRickC137** [comment]
True, we know that, but we give users a chance to reproduce the results of the load_stage anywhere. 

I will fish for a while, maybe inside the big machine of WHO sits a medical terminologist who would be happy to help us and make ICD10 available for downloads. Again.
